build/lib/util.ts

```diff
@@ -415,6 +415,13 @@ export function acquireWebNodePaths() {
 		nodePaths[key] = entryPoint;
 	}
 
+	// @TODO lramos15 can we make this dynamic like the rest of the node paths
```

Comment:
I'm not seeing anything specific in the 1DS package.json, so I'm not sure why this is occurring...

Reply:
Does the AI JS SDK bundle its dependencies while 1DS does not?